ansible.cfg

```diff
@@ -0,0 +1,3 @@
+[defaults]
+interpreter_python      = auto_silent
+log_path                = /var/log/cray/ansible.log
```

Comment:
Can we take **ANSIBLE_FORCE_COLOR=1 PYTHONUNBUFFERED=1 ANSIBLE_STDOUT_CALLBACK=debug**
out of the packer config and add the equivalent here too?
Definitely no reason to hold up merging for it though.